lib/services/rds.js

```diff
@@ -13,11 +13,19 @@ var AWS = require('../core');
      if (crossRegionOperations.indexOf(request.operation) !== -1 &&
          request.params.SourceRegion) {
        request.params = AWS.util.copy(request.params);
-       if (request.params.PresignedUrl ||
+       if (request.params.PreSignedUrl ||
            request.params.SourceRegion === this.config.region) {
          delete request.params.SourceRegion;
        } else {
+         var doesParamValidation = !!this.config.paramValidation;
+         // remove the validate parameters listener so we can re-add it after we build the URL
+         if (doesParamValidation) {
+           request.removeListener('validate', AWS.EventListeners.Core.VALIDATE_PARAMETERS);
```

Comment:
Agreed, we do this often enough throughout the SDK that we should provide a better way to handle this.